af/fastpath/fastpath/tests/test_functional.py

```diff
@@ -23,20 +23,22 @@
 # The fixtures download cans from S3 to a local directory
 #
 # Use credentials from ~/.aws/config in the block:
-# [ooni-data-private]
+# [ooni-data]
 # aws_access_key_id = ...
 # aws_secret_access_key = ...
 #
 # Explore bucket from CLI:
 # AWS_PROFILE=ooni-data-private aws s3 ls s3://ooni-data-private/canned/2019-07-16/
 
-BUCKET_NAME = "ooni-data-private"
+# TODO: drop the boto3 code and use only s3feeder
+BUCKET_NAME = "ooni-data"
```

Comment:
Since you are using the ooni-data bucket, you don't need any access keys anymore. You should look into making all boto requests using unsigned requests mode.

This can be done via:
```
>>> import boto3
>>> from botocore import UNSIGNED
>>> from botocore.config import Config
>>> s3 = boto3.client('s3', config=Config(signature_version=UNSIGNED))
```

Reply:
Migrated this to: https://github.com/ooni/backend/issues/279